### ISSUE TITLE
False Positive on Windows Defender

### ISSUE THREAD

**Issue by Jev1337** [trustedmercury/valorant-store-scanner#1]
![image](https://user-images.githubusercontent.com/19759761/146994322-fbdef58c-1f2c-4349-86df-e9626c510c29.png)
I love the idea behind this. There is one thing that may bother people here which is the false positive.

It would be great if you find a work-arround and find the problem that is causing this false positive.

I personally haven't downloaded it yet and haven't tried it.

**Comment by kevinjosethomas** [comment]
This program isn't even downloadable yet. Additionally, this repository is a website and doesn't need anything to be downloaded so it's impossible for it to trigger Windows Defender.

I'm afraid it's being triggered by another application - and it's extremely likely that it isn't a FALSE positive.

**Comment by Jev1337** [comment]
Hey!
The false positive was on the entire file, sorry to let you know just now, it's because I received the mail of you closing this just now.
When I downloaded this, I downloaded it in offline mode (as in: Downloaded it from another pc) and for some reasons it kept denying the entire folder. I went online and it no longer detects it.

Also Windows Defender tends to make false positives and not detect more viruses when it's offline (https://www.youtube.com/watch?v=rLJZcDIAes4)
